test/unit/math/prim/mat/fun/add_diag_test.cpp

```diff
@@ -59,29 +59,29 @@ TEST(MathPrimMat, double_mat_double_rvec_add_diag) {
 }
 
 TEST(MathPrimMat, var_mat_double_vec_add_diag) {
-  Eigen::Matrix<stan::math::var, Eigen::Dynamic, Eigen::Dynamic> mat(2, 3);
+  Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> mat(2, 3);
   mat << 1, 1, 1, 1, 1, 1;
 
   Eigen::Matrix<double, 1, -1> to_add(2);
```

Comment:
This is an Eigen::RowVectorXd (so the test is mis-labeled as well). I guess there's plenty of places in the code where we use -1 in place of Eigen::Dynamic, but that's a magic constant we shouldn't be abusing.